app/controllers/content_item_signups_controller.rb

```diff
@@ -83,8 +82,7 @@ def handle_redirects
 
   def validate_document_type
```

Comment:
Minor: we could simplify the number of `before_actions` and combine these into a single `validate_signup_params` action, or something.